.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+<!---
```

Comment:
I believe just two `-`s are enough to make it a multi-line comment 😛
Similarly for the issue template.